test_routes.py

```diff
@@ -165,6 +165,69 @@ def test_edit(self):
             self.assertIsNotNone(updated_user)
             self.assertEqual(updated_user.fav_type, 'nano')
 
+
+    def test_forgot_password(self):
```

Comment:
It's more verbose but try to make the test cases more broken down.

Reply:
done